tests/widget/modal/widget-modal.register.test.js

```diff
@@ -17,6 +17,14 @@ test('modal widget', async ({ page }) => {
   await page.fill('text=Password', 'j56eKtae*1');
   await page.selectOption('select', '0');
   await page.fill('text=Security Answer', 'Red');
+
+  const termsLink = page.locator('text=View full Terms & Conditions');
+  const termsUrl = await termsLink.getAttribute('href');
+  const termsTarget = await termsLink.getAttribute('target');
+
+  expect(termsUrl).toBe('https://www.forgerock.com/terms');
+  expect(termsTarget).toBe('_blank');
```

Comment:
On the review call for this PR, we discussed approaches for verifying that the terms link **a)** is correct / as configured and **b)** opens in a new tab. I originally set out to use Playwright to actually click the link and verify that this opens in a new tab. However after discussing this with the team, the consensus seems to be that the value in the e2e test is verifying that the URL is displayed as configured and _should,_ open in a new tab, rather than that the link actually opens in a new tab. I personally think this approach is sufficient, but welcome feedback.